Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
I think `ansible` is basically getting re-sorted in some different way every time since the resource blocks are not actually the result of `poet -f ansible`.